docs/config/palette.md

```diff
@@ -32,6 +32,15 @@ $MQMobileNarrow = 419px
 // code
 $lineNumbersWrapperWidth = 3.5rem
 $codeLang = js ts html md vue css sass scss less stylus go java c sh yaml py docker dockerfile makefile
+
+// fonts
+$fontFamily = 'PT Serif', Serif
+```
+
+For overridden font family, you many need to import the font in order to render proplery. Here is an example of using a Google font:
+```stylus
+@import url('https://fonts.googleapis.com/css2?family=Noto+Serif&family=PT+Serif&display=swap');
+$fontFamily = 'PT Serif', Serif
```

Comment:
Let's say you use Roboto:
```
@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
$fontFamily = 'Roboto', sans-serif
```

Does `PT Serif` font will still be imported and processed by webpack?
I'm still seeing `@require './font.styl'` in https://github.com/bongnv/vuepress-theme-blog/blob/master/styles/index.styl#L1, and file [`font.styl`](https://github.com/bongnv/vuepress-theme-blog/blob/master/styles/font.styl) always exist.

Reply:
```stylus
if $fontFamily is 'PT Serif, Serif'
  @require './font.styl'
```

Reply:
I tried to play around with that but it didn't work. Looks like `$fontFamily` is not available that time. Tried both:
```
if $fontFamily is 'PT Serif, Serif'
  @require './font.styl'
```
and
```
if $fontFamily == 'PT Serif, Serif'
  @require './font.styl'
```